Geometry/TrackerNumberingBuilder/plugins/TrackerGeometricDetESModule.cc

```diff
@@ -24,9 +24,30 @@ using namespace edm;
 
 TrackerGeometricDetESModule::TrackerGeometricDetESModule( const edm::ParameterSet & p ) 
   : fromDDD_( p.getParameter<bool>( "fromDDD" )),
-    layerNumberPXB_( p.exists( "layerNumberPXB" ) ? p.getParameter<unsigned int>( "layerNumberPXB" ) : 16U ),// 16 for current, 18 for SLHC 
-    totalBlade_( p.exists( "totalBlade" ) ? p.getParameter<unsigned int>( "totalBlade" ) : 24U )             // 24 for current, 56 for SLHC 
```

Comment:
The HLT tests running the HLT the "online" way (fully self-contained cfg from ConfDB).
This is tested as part of the special HLT addOn tests, but not in the normal cmsDriver-
based IB tests.